repo/packages/M/minio/0/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png",
+    "icon-medium": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png",
+    "icon-large": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "minio-docker-RELEASE": "minio/minio:latest"
```

Comment:
Issue with `latest` is that it is usually not what people think it is. it may, but it doesnt have to. And it is particularly bad when used on top of our versioning. Ex: this would provide some context: https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375#.xsgt5nipv